### PR TITLE
Fix #11: CDN

### LINKED ISSUE #11: CDN Solution
We know that we'll want some sort of CDN solution for people who want to pull in the design library whole-hog. We'll need to understand:

- [ ] Is this just CSS, or also JS? Are those shipped separately?
- [ ] How is react code vs. vanillaJS shipped in the CDN?

### PR DESCRIPTION
**Pull request by helenvholmes**
Issue number: #11

## **This PR does the following:**
- Adds CDN information to the README

### CODE REVIEW

**Comment by hokei** [comment]
do we have a site anywhere where this is demonstrated?  If it breaks, is there something that will alert us in an automated way?

File: README.md [hunk 1]
```diff
@@ -62,6 +62,18 @@ function NewComponent(props) {
 }
 ```
 
+### CDN
```
Comment (hokei) [inline README.md:65]: 😮 there doesn't need to be code changes? it's just ... here? I never knew!

Comment (helenvholmes) [reply README.md:65]: Neither did I, Edwin completely to the rescue. Now we know!

**Comment by helenvholmes** [comment]
> do we have a site anywhere where this is demonstrated? If it breaks, is there something that will alert us in an automated way?

@hokei We don't have a site where this is demonstrated, but we could! I'm spinning something up on Codepen. I'm not done with it but I'll push up reference to it in this PR and ping you when it's ready.

Here is the site for unpkg: https://unpkg.com/
I haven't gone much deeper than that to figure out what happens if it breaks. We could make reference in the README that it's an at-your-own-risk situation since a third-party will be serving up assets for your site, but I think that's a given for any CDN-type situation. The internet's a pile of poorly stacked irregular rocks at the best of times

**Comment by helenvholmes** [comment]
I'm going to close this out until someone is ready to take a closer look at getting the CDN working with something like Codepen/a github.io static site again.

**Pull request closed without merging**